docs/3.0/select-all.md

```diff
@@ -0,0 +1,43 @@
+# Select All
+
+The "Select All" feature is designed to enable users to select all queried records and perform actions on the entire selection. This feature is particularly useful when dealing with large datasets, allowing users to trigger actions on all queried records, not just the ones visible on the current table page.
+
+## How It Works
+
+<Image src="/assets/img/3_0/select_all.gif" width="687" height="289" alt="Select all demonstration" />
+
+When the "Select All" feature is activated, the actions system needs to handle the entire set of records that matches the user's query, even if only a subset of the records is currently displayed. To achieve this, the entire query is serialized and passed along with the action request.
```

Comment:
```suggestion
When a user toggles the "Select all" checkbox, Avo will first check to see if there are more records than just those displayed on that page, and if there are, it will ask if the user if they want to select all the records or not.

This is being done through serializing the query to be unserialized back in the action.
```